src/psyclone/tests/psyGen_test.py

```diff
@@ -270,7 +270,7 @@ def test_invalid_name():
 def test_valid_return_object_from_name():
     ''' check get_trans_name method return the correct object type '''
     trans = TransInfo()
```

Comment:
pylint complains that `Routine` is imported but unused.

Reply:
Removed